Source/Core/Core/DSP/DSPCore.cpp

```diff
@@ -147,8 +147,8 @@ bool DSPCore_Init(const DSPInitOptions& opts)
 
   std::fill(std::begin(g_dsp.reg_stack_ptr), std::end(g_dsp.reg_stack_ptr), 0);
 
-  for (size_t i = 0; i < ArraySize(g_dsp.reg_stack); i++)
-    std::fill(std::begin(g_dsp.reg_stack[i]), std::end(g_dsp.reg_stack[i]), 0);
+  for (auto& i : g_dsp.reg_stack)
+    std::fill(std::begin(i), std::end(i), 0);
```

Comment:
I dunno about using `i` for a loop value. Granted, in this case I'm not sure what a better name would be, but that just gives the wrong impression when glancing over the code.

Reply:
That'd be the only shortcomings I see with this PR - most of those single-letter iteration variables could use a better name. Or least don't call them `i` which is the most likely one to be used in a plain for loop as counter.